src/tracks/Pattern.cpp

```diff
@@ -1196,15 +1198,11 @@ void PatternView::paintEvent( QPaintEvent * )
 
 	p.setFont( pointSize<8>( p.font() ) );
 
-	QColor text_color = ( m_pat->isMuted() || m_pat->getTrack()->isMuted() )
-		? QColor( 30, 30, 30 )
-		: textColor();
-
 	if( m_pat->name() != m_pat->instrumentTrack()->name() )
 	{
-		p.setPen( QColor( 0, 0, 0 ) );
+		p.setPen( textShadowColor() );
 		p.drawText( 4, p.fontMetrics().height()+1, m_pat->name() );
-		p.setPen( text_color );
```

Comment:
Well, on some patterns the text changed the color when it was muted, and on some it remained the same. The text got pretty unreadable on muted patterns, and I think that's undesired, so I made the text always the same color, without looking if the pattern is muted or not. I noted it in the PR description: 
> 6. All pattern text stays the same color when the pattern is muted, for better readability.